pkgs/top-level/all-packages.nix

```diff
@@ -33413,6 +33413,8 @@ with pkgs;
 
   linuxsampler = callPackage ../applications/audio/linuxsampler { };
 
+  litterbox = callPackage ../applications/networking/irc/litterbox { };
+
```

Comment:
New packages are recommended to go to `pkgs/by-name` https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md

```suggestion

```

Reply:
Ah, sorry about that. I got caught up in doing it a way symmetric with `pounce`. I have corrected this and re-tested to satisfaction.